lms/djangoapps/courseware/views/views.py

```diff
@@ -1583,6 +1608,18 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
                     )
                 )
 
+        # For courses using an LTI provider managed by edx-exams:
+        # Access to exam content is determined by edx-exams and passed to the LMS using a
+        # JWT cookie. There is no longer a need for exam gating or logic inside the
```

Comment:
```suggestion
        # JWT token. There is no longer a need for exam gating or logic inside the
```

Reply:
It would be helpful to have the comment explicitly spell out that the JWT is passed in via query param